capabilities.tf

```diff
@@ -1,6 +1,9 @@
 // This file is replaced by code-generation using 'capabilities.tf.tmpl'
 // This file helps app module creators define a contract for what types of capability outputs are supported.
 locals {
+
+  cap_env_vars = {}
+  cap_secrets  = {}
```

Comment:
@BSick7 - This looks like the right place to add these in order to help define the structure for the IDE. I think you have the base implementation working so that I don't really have to do any other work here, right?

Reply:
This is correct. `capabilities.tf` is generated from `capabilities.tf.tmpl` before running the plan.
The declaration of this local is defined: https://github.com/nullstone-modules/aws-beanstalk-app/blob/master/capabilities.tf.tmpl#L56-L60